fyle/platform/apis/v1beta/admin/reimbursements.py

```diff
@@ -6,9 +6,11 @@
 from ....internals.list_all_resources import ListAllResources
 from ....internals.list_resources import ListResources
 from ....internals.post_resources import PostResources
+from ....internals.post_bulk_resources import PostBulkResources
 
 
-class Reimbursements(ListResources, ListAllResources, PostResources, GetResources):
+
+class Reimbursements(ListResources, ListAllResources, PostResources, GetResources, PostBulkResources):
```

Comment:
for reimbursements to do POST BULK, the endpoint changes 
`/admin/reimbursements/mark_paid/bulk`

Can we add a separate method for reimbursements

Reply:
okay will fix this